var/spack/repos/builtin/packages/py-humanize/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHumanize(PythonPackage):
+    """Python package that contains humanization utilities"""
+
+    homepage = "https://github.com/jmoiron/humanize"
+    url      = "git@github.com:jmoiron/humanize.git"
+
+    version('develop', git=url, branch='master', preferred=True)
+    version('0.5.1', git=url, tag='0.5.1')
```

Comment:
preferred version should be released one and not the branch which is constantly changing